packages/diagram/src/module/diagrams/content/uml/nonNavigableAssociations.ts

```diff
@@ -0,0 +1,35 @@
+import { InterpreterModule, parse } from "@hylimo/core";
+
+/**
+ * Module providing associations without navigation operators
+ */
+export const nonNavigableAssociationsModule = InterpreterModule.create(
+    "uml/nonNavigableConnections",
+    ["common/defaultMarkers"],
+    [],
+    [
+        ...parse(
+            `
+                this.create = scope.internal.createConnectionOperator
+                scope.!-- = create(
+                    startMarkerFactory = scope.defaultMarkers.cross
+                )
+                scope.--! = create(
+                    endMarkerFactory = scope.defaultMarkers.cross
+                )
+                scope.!--! = create(
```

Comment:
WTF is this abomination?
Who wants an association that is inaccessible in both directions?

Reply:
idk, probably not that useful
I created it primarily due to "why not" back then